bot/handlers.py

```diff
@@ -410,15 +349,14 @@ def fallbacks_handler(
 
     def get_conversation_handler(self,) -> ConversationHandler:
         conv_handler = ConversationHandler(
-            entry_points=[CallbackQueryHandler(self.callback_query_handler),
-                          CommandHandler('show_debts', self.show_debts),
-                          CommandHandler('add_expense', self.add_expense), ],
+            entry_points=[CallbackQueryHandler(self.callback_query_handler), ],
             states={
                 States.EXPENSE_NAME: [AddExpenseHandlers().get_conversation_handler()],
             },
             fallbacks=[MessageHandler(Filters.all, self.fallbacks_handler)],
             map_to_parent={
-                ConversationHandler.END: ConversationHandler.END
+                ConversationHandler.END: ConversationHandler.END,
+                States.EVENT_ACTIONS: States.EVENT_ACTIONS
```

Comment:
Please add a single comma to the end, like in previous line

Reply:
Done